app/models/user.rb

```diff
@@ -10,7 +10,7 @@ class User < ApplicationRecord
   def self.from_omniauth(auth)
     where(provider: auth.provider, uid: auth.uid).first_or_create do |user|
       user.provider = auth.provider
-      user.uid      = auth.uid
+      user.uid      = auth.info.nickname
```

Comment:
maybe should create a migration to rename this column to `login/nickname/username` because uid works unpredictable a little but with username you can be detached from octokit context, it will also work, but since you will decide to use another gem or pure api for this, it will be easier to migrate and stay gem agnostic.